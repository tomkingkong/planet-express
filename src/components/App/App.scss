.App {
  display: flex;
  flex-direction: column;
  position: relative;
  min-height: 100vh;
  height: 100%;
}