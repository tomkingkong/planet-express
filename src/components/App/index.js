import React, { Component } from 'react';

import './App.css';
import { Routes } from '../Routes';
import Fetch from '../../containers/Fetch';

class App extends Component {
  render() {
    return (
      <div className="App">
        <Fetch />
        <Routes />
      </div>
    );
  }
}

export default App;
