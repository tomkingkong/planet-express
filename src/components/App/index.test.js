import React from 'react';
import App from '.';
import { shallow } from 'enzyme';


describe('App component', () => {  
  it('should match snapshot', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
})
