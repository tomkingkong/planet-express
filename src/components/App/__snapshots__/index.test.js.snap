// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should match snapshot 1`] = `
<div
  className="App"
>
  <Connect(Fetch) />
  <Component />
  <Component />
</div>
`;
